packages/frontend-shared/src/gql-components/HeaderBarContent.cy.tsx

```diff
@@ -282,7 +279,6 @@ describe('<HeaderBarContent />', { viewportWidth: 1000, viewportHeight: 750 }, (
     cy.contains(cloudViewer.fullName).should('be.visible')
     cy.contains(cloudViewer.email).should('be.visible')
     cy.findByRole('button', { name: text.login.actionLogout }).should('be.visible')
-    cy.percySnapshot()
```

Comment:
I added an `afterEach` hook with `cy.percySnapshot` in this file, so anywhere there was a snapshot at the end of a test I removed it